spec/factories/applicants.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
```

Comment:
I didn't know much about Factory Bot, but what a cool tool! I will research it more so I can implement it in my own projects in the future!